site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,18 +16,79 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
-### 2021.06.08: Introducing chrome.scripting (blog post)
+### New "Introducing chrome.scripting" blog post
+#### Published 2021.06.08
 
 The [Scripting API](/docs/extensions/reference/scripting/) is a new Manifest V3 API focused on,
 well, scripting. In this post we dig into the motivations for this change and take a closer look at
 some of the new capabilities it introduces. [Read the post](/blog/crx-scripting-api).
 
-### 2021.01.19: Manifest V3 launched
+### ES modules for service workers
+#### Launched in Chrome 91
+
+JavaScript supports modules in service workers. In your manifest, specify a module in your manifest:
+
+```js
+"background": {
+  "service_worker": "script.js",
+  "type": "module"
+}
+```
+
+This loads the worker script as an ES module, so you can import it in other worker
+scripts.
```

Comment:
```suggestion
This loads the worker script as an ES module, so you can use the `import` keyword in the worker script.
```

I think this is more relevant, but feel free to run it by Simeon, too

Reply:
Hm, I think a slightly different wording would be better. 

```suggestion
This loads the worker script as an ES module, which allows you to use the `import` keyword in the
worker's script to import other modules.
```

My suggestion feels a little clunky to me; the key part I wanted to get across is that declaring your service worker as a module allows you to import module scripts. If you don't specify `"type": "module"` in the manifest, the service worker is loaded as a classic script and cannot use dynamic import (`import("./file.js").then((module) => { self.myMethod = module.myMethod })`) or static import (`import { myMethod } from "./file.js";`) in your service worker.